Hunting/Hunter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hunting
+{
+    public class Hunter
+    {
+        private Hare hare;
+
+        public Hunter(Hare hare)
```

Comment:
Not good approach to aggregate hare with hunter.
It would be better to store them separately.